pkg/events/parse/params.go

```diff
@@ -45,6 +45,19 @@ func ArgUint64Val(event *trace.Event, argName string) (uint64, error) {
 	return 0, fmt.Errorf("argument %s not found", argName)
 }
 
+func ArgInt64Val(event *trace.Event, argName string) (int64, error) {
```

Comment:
@AsafEitani would you be able to add test coverage to these functions? Or perhaps, if you are busy, mind creating an issue to add coverage here and tag with "good first issue"?